.github/workflows/test-deploy-wf-service.yml

```
@@ -25,16 +21,13 @@ on:
-      sha:
-        type: string
-        description: 'Commit SHA ID'
-        required: true
-        default: ''
+
 
 env:
   REGISTRY: ghcr.io
   IMAGE_NAME: ${{ github.repository_owner }}/workflows-service
-  SHORT_HASH: ''
+  SHORT_HASH: ${{ inputs.environment == 'prod' && github.env.SB_WF_SHORT_SHA || github.env.DEV_WF_SHORT_SHA }}
```

actionlint

receiver of object dereference "sb_wf_short_sha" must be type of object but got "string"

(expression)